charts/operator-wandb/values.yaml

```diff
@@ -48,6 +48,12 @@ global:
   slack:
     secret: ""
     clientId: ""
+  
+  email:
```

Comment:
maybe

```
email:
  stmp:
```

We might had support for other protocals

Reply:
`pop`, `imap`, and `smtp` for completeness